### PR TITLE
Review of Operators (part 1)

### PR DESCRIPTION
**Pull request by eksperimental**
This is part one (of two hopefully), broken into several commits to ease the reviewing

### CODE REVIEW

File: lib/elixir/pages/Operators.md [hunk 1]
```diff
@@ -67,12 +69,20 @@ The reason we can compare different data types is pragmatism. Sorting algorithms
 number < atom < reference < function < port < pid < tuple < map < list < bitstring
 ```
 
-When comparing two numbers of different types (a number is either an integer or a float), a conversion to the type with greater precision will always occur, unless the comparison operator used is either `===` or `!==`. A float will be considered more precise than an integer, unless the float is greater/less than +/-9007199254740992.0, at which point all the significant figures of the float are to the left of the decimal point. This behavior exists so that the comparison of large numbers remains transitive.
+When comparing two numbers of different types (a number being either an integer
+or a float), a conversion to the type with greater precision will always occur,
+unless the comparison operator used is either [`===`](`===/2`) or [`!==`](`!===/2`).
+A float will be considered more precise than an integer, unless the float is
+greater/less than `+/-9007199254740992.0` respectively, at which point all the
```
Comment (eksperimental) [inline lib/elixir/pages/Operators.md]: backticks added to `+/-9007199254740992.0`
"respectively" added for clarity

File: lib/elixir/pages/Operators.md [hunk 2]
```diff
@@ -67,12 +69,20 @@ The reason we can compare different data types is pragmatism. Sorting algorithms
...
+exists so that the comparison of large numbers remains transitive.
 
 The collection types are compared using the following rules:
 
-* Tuples are compared by size then element by element.
-* Maps are compared by size then by keys in ascending term order then by values in key order. In the specific case of maps' key ordering, integers are always considered to be less than floats.
+* Tuples are compared by size, then element by element.
+* Maps are compared by size, then by keys in ascending term order, then by values
+  in key order. In the specific case of maps' key ordering, integers are always
+  considered to be less than floats.
```
Comment (eksperimental) [inline lib/elixir/pages/Operators.md]: comas added before "then"

File: lib/elixir/pages/Operators.md (hunk 1 above)
Comment (fertapric) [reply lib/elixir/pages/Operators.md]: Why the backticks?

File: lib/elixir/pages/Operators.md [hunk 3]
```diff
@@ -67,12 +69,20 @@ The reason we can compare different data types is pragmatism. Sorting algorithms
 number < atom < reference < function < port < pid < tuple < map < list < bitstring
 ```
 
-When comparing two numbers of different types (a number is either an integer or a float), a conversion to the type with greater precision will always occur, unless the comparison operator used is either `===` or `!==`. A float will be considered more precise than an integer, unless the float is greater/less than +/-9007199254740992.0, at which point all the significant figures of the float are to the left of the decimal point. This behavior exists so that the comparison of large numbers remains transitive.
+When comparing two numbers of different types (a number being either an integer
```
Comment (josevalim) [inline lib/elixir/pages/Operators.md]: We don't break text in our guides line by line like this, so please revert this change and keep everything as a long line. I would honestly prefer to have it line by line but this is not the time to change it. :)

File: lib/elixir/pages/Operators.md (hunk 1 above)
Comment (eksperimental) [reply lib/elixir/pages/Operators.md]: True. They shouldn't have been used in this case.

**Comment by eksperimental** [comment]
suggestions implemented!

File: lib/elixir/pages/Operators.md [hunk 4]
```diff
@@ -67,12 +68,12 @@ The reason we can compare different data types is pragmatism. Sorting algorithms
 number < atom < reference < function < port < pid < tuple < map < list < bitstring
 ```
 
-When comparing two numbers of different types (a number is either an integer or a float), a conversion to the type with greater precision will always occur, unless the comparison operator used is either `===` or `!==`. A float will be considered more precise than an integer, unless the float is greater/less than +/-9007199254740992.0, at which point all the significant figures of the float are to the left of the decimal point. This behavior exists so that the comparison of large numbers remains transitive.
+When comparing two numbers of different types (a number being either an integer or a float), a conversion to the type with greater precision will always occur, unless the comparison operator used is either [`===`](`===/2`) or [`!==`](`!===/2`). A float will be considered more precise than an integer, unless the float is greater/less than +/-9007199254740992.0 respectively, at which point all the significant figures of the float are to the left of the decimal point. This behavior exists so that the comparison of large numbers remains transitive.
```
Comment (fertapric) [inline lib/elixir/pages/Operators.md]:
```suggestion
When comparing two numbers of different types (a number being either an integer or a float), a conversion to the type with greater precision will always occur, unless the comparison operator used is either [`===`](`===/2`) or [`!==`](`!==/2`). A float will be considered more precise than an integer, unless the float is greater/less than +/-9007199254740992.0 respectively, at which point all the significant figures of the float are to the left of the decimal point. This behavior exists so that the comparison of large numbers remains transitive.
```

The link had an extra `=` (`` [`!==`](`!===/2`) ``)

Comment (fertapric) [reply lib/elixir/pages/Operators.md]: And maybe use `Kernel.` like the links above?

Comment (eksperimental) [reply lib/elixir/pages/Operators.md]: Im planing another PR removing all mentions to Kernel and Kernel.SpecialForms

Comment (eksperimental) [reply lib/elixir/pages/Operators.md]: link to `!==/2` has been fixed

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:
